skill-tree /.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI Build
+
+on:
+  pull_request:
+    branches:
+      - '**'
+jobs:
+  build:
+    name: Maven Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Java 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
+          cache: 'maven'
```

Comment:
- Do we want to add an architecture (ex: x64)?

Reply:
The GitHub actions runner by default uses x86 runner, for the ARM runner we will have to setup our own runner 
Ref: https://github.com/orgs/community/discussions/25319